packages/office-ui-fabric-react/src/components/Persona/PersonaCoin.tsx

```diff
@@ -89,16 +91,22 @@ export class PersonaCoin extends React.Component<IPersonaProps, IPersonaState> {
 
     let size = this.props.size as PersonaSize;
     let divProps = getNativeProps(this.props, divProperties);
+    let coinSizeStyle = coinSize ? { width: coinSize, height: coinSize } : undefined;
 
     initialsColor = initialsColor !== undefined && initialsColor !== null ? initialsColor : this._getColorFromName(primaryText);
 
     return (
       <div
         { ...divProps }
         className={ css('ms-Persona', styles.root, className, PERSONA_SIZE[size], PERSONA_PRESENCE[presence as PersonaPresenceEnum], this.props.showSecondaryText ? styles.showSecondaryText : '') }
+        style={ coinSize ? { height: coinSize, minWidth: coinSize } : undefined }
```

Comment:
Please use mergeStyles instead of inline styles. Here's an example

https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/Tooltip/Tooltip.tsx#L47

feel free to ping me if you have questions about MergeStyles function

Reply:
I don't think it's necessary here. If you're explicitly setting a size that could be dynamicly changing there's just about no value in using mergeStyles.

Reply:
I'm a bit concerned about the deal with `presence` component in Persona. We cannot deal with the place of it properly for custom coin size as it uses absolute position.
I will make further trial on this part.